packages/core/src/components/BorderedList/Item.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import withStyles, { css, WithStylesProps } from '../../composers/withStyles';
+
+export type Props = {
+  /** Item content. */
+  children: NonNullable<React.ReactNode>;
+  /** Render with reduced vertical padding. */
+  compact?: boolean;
```

Comment:
Let's add that prop type that these 2 can't be used at the same time.

Reply:
updated!